sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -1235,26 +1235,32 @@ class DockerSdkWorkerHandler(GrpcWorkerHandler):
   def __init__(self, payload, state, provision_info):
     super(DockerSdkWorkerHandler, self).__init__(state, provision_info)
     self._container_image = payload.container_image
+    self._options = payload.options
     self._container_id = None
 
   def start_worker(self):
     try:
       subprocess.check_call(['docker', 'pull', self._container_image])
     except Exception:
       logging.info('Unable to pull image %s' % self._container_image)
-    self._container_id = subprocess.check_output(
-        ['docker',
-         'run',
-         '-d',
-         # TODO:  credentials
-         '--network=host',
-         self._container_image,
-         '--id=%s' % uuid.uuid4(),
-         '--logging_endpoint=%s' % self.logging_api_service_descriptor().url,
-         '--control_endpoint=%s' % self.control_address,
-         '--artifact_endpoint=%s' % self.control_address,
-         '--provision_endpoint=%s' % self.control_address,
-        ]).strip()
+    cmd = [
+      'docker',
+      'run',
+      '-d',
+      # TODO:  credentials
+      '--network=host',
+    ]
+    if self._options:
+      cmd.extend(self._options.split(' '))
+    cmd.extend([
+      self._container_image,
+      '--id=%s' % uuid.uuid4(),
+      '--logging_endpoint=%s' % self.logging_api_service_descriptor().url,
+      '--control_endpoint=%s' % self.control_address,
+      '--artifact_endpoint=%s' % self.control_address,
+      '--provision_endpoint=%s' % self.control_address,
+    ])
+    self._container_id = subprocess.check_output(cmd).strip()
```

Comment:
As far as I can see this is just reformating? Please consider reverting since this is unrelated.

Reply:
The reformatting was necessary to add the optional `DockerPayload.options` into the docker run command.

```python
    if self._options:
      cmd.extend(self._options.split(' '))
```

I didn't see anywhere this class was being used... I'm happy to revert if we don't want/need this functionality here.